Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -501,7 +501,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        return empty($changed) ? 0 : 1;
+        return $resolver->isDryRun() || empty($changed) ? 0 : 2;
```

Comment:
Btw, the commit message describes the expected behavior. but the change done inside the commit does not correspond to the commit message